### PR TITLE
Update CI and flake

### PR DESCRIPTION
**Pull request by ThetaSinner**

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
```
Comment (steveej) [inline setup.sh:1]: accidental commit of this file?

Comment (ThetaSinner) [reply setup.sh]: Yes, was checking the Nix version, removed!

**Comment by steveej** [comment]
what's missing here is a writable nix cache. at least the dependencies might not change on every PR.

**Comment by ThetaSinner** [comment]
Definitely agree with that. It'd speed things up significantly. So would the suggestion to split the test script to run separate tests in parallel. I think this is enough for one PR and we should look at caching separately

**Review by steveej — APPROVED** [review]

**Comment by robbiecarlton** [comment]
Closing, this work has been incorporated into holochain-0.1 and holochain-0.2 branches

**Pull request closed without merging**